app/datasets/cfpb_companies.json

```diff
@@ -0,0 +1 @@
tection Association, L.P."},{"name":"Certified Credit Reporting, Inc."},{"name":"SECURITY FINANCE CORP"},{"name":"JH Portfolio Debt Equities, LLC"},{"name":"Selene Finance LP"},{"name":"FIRST NATIONAL BANK OF OMAHA"},{"name":"The CMI Group, Inc."},{"name":"Enova International, Inc."},{"name":"FAIR COLLECTIONS \u0026 OUTSOURCING, INC."},{"name":"Manley Deas \u0026 Kochalski LLC"},{"name":"Nelson Cruz \u0026 Associates LLC"},{"name":"Credit Karma, Inc."},{"name":"Kings Credit Services"},{"name":"VW Credit"},{"name":"EARLY WARNING SERVICES, LLC"},{"name":"Monarch Recovery Holdings, Inc."},{"name":"TOWER LOAN OF MISSISSIPPI, INC."},{"name":"National Credit Adjusters, LLC"},{"name":"FIFTH THIRD FINANCIAL CORPORATION"},{"name":"EMBRACE HOME LOANS"},{"name":"Century Debt Solutions Inc."},{"name":"ERC"},{"name":"FC HoldCo LLC"},{"name":"HOME FINANCING CENTER"},{"name":"HYUNDAI CAPITAL AMERICA"},{"name":"McCarthy, Burgess \u0026 Wolff, Inc."},{"name":"Specialized Loan Servicing LLC"},{"name":"LEXISNEXIS"},{"name":"Critical Resolution Mediation LLC"},{"name":"Innovis"},{"name":"FIRST CITIZENS BANCSHARES, INC."},{"name":"M\u0026T BANK CORPORATION"},{"name":"First Investors Financial Services Group, Inc."},{"name":"Collections Acquisition Company, Inc."},{"name":"SOUTHERN MANAGEMENT CORP."},{"name":"Consumer Collection Management. Inc"},{"name":"First Credit Services Inc."},{"name":"HireRight Solutions, Inc."},{"name":"Zenith Financial Network Inc"},{"name":"RUSHMORE LOAN MANAGEMENT SERVICES LLC"},{"name":"Kenneth, Eisen \u0026 Associates, Ltd."},{"name":"EXETER FINANCE CORP."},{"name":"Worldwide Recovery LLC"},{"name":"GLA Collection Company, Inc."},{"name":"ALG Trustee LLC"},{"name":"Tiburon Financial, LLC"},{"name":"BAYVIEW LOAN SERVICING, LLC"},{"name":"Adler Wallach \u0026 Associates, Inc."},{"name":"Amsher Collection Services, Inc."},{"name":"First Associates Loan Servicing LLC"},{"name":"Nationwide Recovery Systems, Ltd"},{"name":"NAM National Arbitration and Mediation"},{"name":"BlueChip Financial"},{"name":"Medical Data Systems, Inc."},{"name":"Nevada Credico, Inc."},{"name":"Waypoint Resource Group, LLC"},{"name":"Mountain Land Collections, Inc."},{"name":"International Computer Systems, Inc."},{"name":"NEW CITY FUNDING CORP."},{"name":"Convergent Resources, Inc."},{"name":"SUMMIT FINANCIAL CORP"},{"name":"Servis One, Inc."},{"name":"ZWICKER \u0026 ASSOCIATES"},{"name":"ProCollect, Inc"},{"name":"Town \u0026 Country Acceptance Corporation"},{"name":"STATE FARM BANK, FSB"},{"name":"Credence Resource Management, LLC"},{"name":"National Collection Systems, Inc."},{"name":"LENDING CLUB CORP"},{"name":"loanDepot.com, LLC"},{"name":"Aldous \u0026 Associates, PLLC"},{"name":"CITY NATIONAL BANK"},{"name":"DFC GLOBAL CORP"},{"name":"Account Control Technology, Inc."},{"name":"United Recovery Solutions"},{"name":"GLOBAL CLIENT SOLUTIONS, LLC"},{"name":"LAKEVIEW LOAN SERVICING, LLC"},{"name":"CARRINGTON MORTGAGE SERVICES, LLC"},{"name":"Envoy Mortgage Ltd."}]
```

Comment:
Not a huge fan of just mashin' the JSON in here, but these are slow calls.

These are refreshable via the rake task.